controller/userController.js

```diff
@@ -0,0 +1,33 @@
+const File = require("../models/fileModel");
+const { v4 :uuidv4 }= require ('uuid');
+
+const fileUploader=async(req, res) => {
+      //validate
+      // console.log(req.file);
+      if (!req.file) {
+        return res.json({ error: "All fields are required" });
```

Comment:
Put error message to constants file

Reply:
I didn't understand what u want to say

do you mean to create new error file and put all error in that file

Reply:
Create such a file in utils/constants.js
```
const USER_CONTROLLER = {
    FILE_UPLOAD_ERROR_MSG: "All fields are required"
}
```

And use this variable instead of writing string in controller.

Reply:
Added utils file